colcon-build-and-test/action.yml

```diff
@@ -0,0 +1,90 @@
+name: "colcon-build-and-test"
```

Comment:
[Nits] Why did you add double quotes? I'd like to know if there are any specific reasons.

Reply:
I saw this document.
https://docs.github.com/en/actions/creating-actions/creating-a-composite-action

Reply:
I understand, but I believe what is important is consistency, so I'll apply yamllint later.

Reply:
0aedfe0d88221bea0e9d036b9320a7553e191c3c